tequilapi/endpoints/proposals_test.go

```diff
@@ -0,0 +1,135 @@
+package endpoints
+
+import (
+	"github.com/mysterium/node/money"
+	"github.com/mysterium/node/server"
+	dto_discovery "github.com/mysterium/node/service_discovery/dto"
+	"github.com/stretchr/testify/assert"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+)
+
+type TestServiceDefinition struct{}
+
+func (service TestServiceDefinition) GetLocation() dto_discovery.Location {
+	return dto_discovery.Location{ASN: "LT"}
+}
+
+type TestPaymentMethod struct{}
+
+func (method TestPaymentMethod) GetPrice() money.Money {
+	return money.Money{}
+}
+
+var proposals = []dto_discovery.ServiceProposal{
+	{
+		Id:                1,
+		Format:            "service-proposal/v1",
```

Comment:
Remove fields, unnecessary for your tests: `Format`, `PaymentMethod`, `PaymentMethodType`, `ProviderContacts`